tools/lib/isTileset.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+var path = require('path');
+
+module.exports = isTileset;
+
+/**
+ * @private
+ */
+function isTileset(file) {
+    var extension = path.extname(file);
+    return extension === '.json';
```

Comment:
Would it be more assertive to check the contents of the file for `asset`, `geometricError` and `root` properties which are all required for tileset.json?

The way it currently is, there is no different between `isJson` and `isTileset`.